docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,518 @@
+.. _worker-selection:
+
+Selection
+=========
+
+Lightly allows you to specify the subset to be selected based on several objectives.
+
+E.g. you can specify that the images in the subset should be visually diverse, be images the model struggles with (Active Learning),
+should only be sharp images, or have a certain distribution of classes, e.g. be 50% from sunny, 30% from cloudy and 20% from rainy weather.
+
+Each of these objectives is defined by a `strategy`. A strategy consists of two parts:
+
+- The :code:`input` defines which data the objective is defined on. This data is either a scalar number or a vector for each sample in the dataset.
+- The :code:`strategy` itself defines the objective to apply on the input data.
+
+Lightly allows you to specify several objectives at the same time. The algorithms try to fulfil all objectives simultaneously.
+
+Lightly's data selection algorithms are supporting four types of input:
+
+- **Embeddings** computed using `our open source framework for self-supervised learning <https://github.com/lightly-ai/lightly>`_
+- **Lightly metadata** are metadata of images like the sharpness and computed out of the images themselves by Lightly.
+- (Optional)  :ref:`Model predictions <docker-datasource-predictions>` such as classifications, object detections or segmentations
+- (Optional) :ref:`Custom metadata <docker-datasource-metadata>` can be anything you can encode in a json file (from numbers to categorical strings)
+
+Prerequisites
+-------------
+
+For using the selection, you must have
+
+- Started the Lightly Worker in worker mode. See :ref:`worker-register`.
+
+- Set up a dataset in the Lightly Platform with a cloud storage as datasource. See :ref:`worker-creating-a-dataset`
+
+Scheduling a Lightly Worker run with selection
+----------------------------------------------
+
+For scheduling a Lightly Worker run with a specific selection,
+you can use the python client and its :py:`schedule_compute_worker_run` method.
```

Comment:
this link doesn't render correctly

Reply:
![image](https://user-images.githubusercontent.com/20324507/188821934-2d51e7bd-35d1-46d5-a804-bfce276f73d6.jpeg)